tests/urls.py

```diff
@@ -3,9 +3,7 @@
 from django.conf.urls.static import static
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 
-from openwisp_utils.admin_theme.admin import admin, openwisp_admin
-
-openwisp_admin()
+from openwisp_utils.admin_theme.admin import admin
```

Comment:
change this to `from django.contrib import admin` and use `isort -y` to reorder the imports